sns/main.tf

```diff
@@ -16,7 +16,7 @@ resource "aws_sns_topic_subscription" "lambda_subscriptions" {
   endpoint             = each.value
   protocol             = "lambda"
   topic_arn            = aws_sns_topic.sns_topic.arn
-  raw_message_delivery = true
+  raw_message_delivery = false
```

Comment:
Just curious, how come we are setting this to false?

Reply:
Raw message delivery isn't supported for lambda endpoints from SNS. https://docs.aws.amazon.com/sns/latest/dg/sns-large-payload-raw-message-delivery.html